### DISCUSSION TITLE
Code editing

### DISCUSSION THREAD

**Discussion by NitinIsFit** [community/community#61574] [Mobile] [answered]
Many years have passed, I am damned sure that GitHub can make a code editor for mobile. It's not even hard for their skilled and talented programming team, but they still don't provide this function to mobile. C'mon GitHub mobile team, what's holding you back?

**Comment by GhostOf0days** [comment]
Why would you code on mobile?

**Reply by NitinIsFit** [reply]
I do not have a computer, do you do crypto? Maybe have a personal PC? or parents gifted you PC?

**Comment by aconsuegra** [comment, marked as answer]
Hi @Nit-nit, thanks for reaching out with your feedback. The GitHub Mobile apps, both on Android and iOS, already support code editing, committing and creating pull requests. You can do it from the Files Changed view inside a pull request, or while browsing code in a repository.

**Reply by NitinIsFit** [reply]
WOAH WAIT WHAT? When did that happen? Damn, I just checked it out right now after receiving your notification, this is good and appreciative, and I hope it gets better in near future, best of luck to the GitHub mobile team

**Reply by NitinIsFit** [reply]
After testing this function I have came to a conclusion of giving you guys some feedback, You should add some settings interface, and existing interface can be improved, searching and replacing for texts, adjustment for text size, colors are necessary because it's GitHub! Congrats to GitHub team again!

**Comment by mesutt000** [comment]
106502a0b2ad8f98b3dbeab47922aad64dd2a30c
